[INTERNAL] Draft of 2020-05-01 sit rep (#385)
nextstrain/ncov: narratives/ncov_sit-rep_2020-05-01.md

<!-- This is left-side text 2-->
# [COVID-19 Resources](https://nextstrain.org/ncov/europe/2020-05-01?d=tree&f_region=Europe&legend=closed)
We've prepared some resources that will make interpreting the data we present in this narrative easier.
#### Nextstrain Resources  
* [START HERE: How to read a phylogeny](https://nextstrain.org/narratives/trees-background/).  
* [Background on coronaviruses](https://nextstrain.org/help/coronavirus/human-CoV).
* [Common misconceptions](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-13?n=11).

#### External Resources  
* [How SARS-CoV-2 mutates and spreads](https://www.nytimes.com/interactive/2020/04/30/science/coronavirus-mutations.html).
* [Ask a Scientist & FAQs](https://covid19.fas.org/).
* [WHO Situation Reports](https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports).
* [CDC Resources](https://www.cdc.gov/coronavirus/2019-ncov/index.html).
* [Why the coronavirus is so confusing](https://www.theatlantic.com/health/archive/2020/04/pandemic-confusing-uncertainty/610819/).
<!-- There is no right-side text -->

<!-- ############ SLIDE BREAK ############# -->

<!-- This is left-side text 3-->

> # [A note on sampling](https://nextstrain.org/ncov/europe/2020-05-01?c=country&r=country&d=map&p=grid&legend=closed)

Reviewer: ```suggestion
# [A note on sampling](https://nextstrain.org/ncov/europe/2020-05-01?c=country&r=country&d=map&f_region=Europe&p=grid&legend=closed)
```
I think this is right, to filter to Europe? I think this will help see the Europe circles better, hopefully, if the zoom on the map goes in a bit!

<!-- There is no right side text -->

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 7-->
# [Five local transmission chains in the Czech Republic](https://nextstrain.org/ncov/europe/2020-04-29?d=tree,map&f_country=Czech%20Republic&m=div&p=full)

The data from the Czech Republic show a mix of these two patterns.
<br><br>
There are at least 7 instances where a sample isolated in the Czech Republic descends from a virus that was mostly likely circulating elsewhere in Europe, indicating new introductions between March 1 and April 2.
<br><br>
For 5 of these introductions, we also see closely related local cases, sampled over time. This is consistent with local transmission.
<br><br>
From this, we can say that there were at least 7 introductions to the Czech Republic; at least 5 of these likely led to further local transmission.

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 8-->

> # [Extensive mixing of cases across Turkey](https://nextstrain.org/ncov/europe/2020-04-29?c=country_exposure&d=tree,map&f_country=Turkey&label=clade:A2&p=full&r=division&legend=open)

Reviewer: ```suggestion
# [Extensive mixing of cases across Turkey](https://nextstrain.org/ncov/europe/2020-04-29?c=country_exposure&d=tree,map&f_country=Turkey&label=clade:A2&p=full&r=division)
```
I'd have legends closed here myself - I found them taking up a lot of the screen - viewing on laptop sized screen.

In the middle of the tree, we see a string of cases sampled from many different provinces around Turkey.
Some of these cases have reported travel history to Saudi Arabia, with subsequent closely related cases in Turkey.
<br><br>
Normally, we might assume that returning travelers had passed the virus to household members or neighbors back home.
Interestingly, though, the travelers' cases were often sampled from different provinces than the most closely related cases.
<br><br>
This indicates that there has been extensive mixing of viral lineages, reaching many provinces of Turkey within one week.
We may also speculate that a large gathering could explain these observations, although we would need additional epidemiological data to further investigate.
<!-- There is no right-side text -->

<!-- There is no right-side text -->
<!-- ############ SLIDE BREAK ############# -->

<!-- This is left-side text 9-->

> # [At least 11 introductions to Poland](https://nextstrain.org/ncov/europe/2020-04-29?d=tree,map&f_country=Poland&p=full)

Reviewer: ```suggestion
# [At least 11 introductions to Poland](https://nextstrain.org/ncov/europe/2020-04-29?d=tree,map&f_country=Poland&p=full)
```
```suggestion
# [At least 11 introductions to Poland](https://nextstrain.org/ncov/europe/2020-04-29?d=tree,map&f_country=Poland&p=full&legend=closed)
```
Is this how to close a legend?

Reviewer: Don't know why this appears twice... sorry.

We have 15 sequences from Poland on Nextstrain. Of these 15, at least 11 of them represent separate introductions from elsewhere in the world.
As in Moscow, given the number of reported cases, we might expect that several of these introductions have seeded local transmission chains in Poland, but we can't yet see this in the genomic data.
<!-- There is no right-side text -->

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 10-->

> # [Early samples from Hungary, Latvia, Slovakia and Slovenia](https://nextstrain.org/ncov/europe/2020-04-29?d=tree,map&f_country=Hungary,Latvia,Slovakia,Slovenia&f_region=Europe&p=full)

Reviewer: ```suggestion
# [Early samples from Hungary, Latvia, Slovakia and Slovenia](https://nextstrain.org/ncov/europe/2020-04-29?d=tree,map&f_country=Hungary,Latvia,Slovakia,Slovenia&f_region=Europe&p=full&legend=closed)
```
Again, I'd close legend on this one

---
title: Genomic analysis of COVID-19 spread. Situation report 2020-05-01.
authors:
  - Sidney Bell  
  - Nicola Müller
  - Cassia Wagner
  - Emma Hodcroft
  - James Hadfield

>   - Sidney M. Bell

Reviewer 2: ```suggestion
  - Sidney M. Bell
```
```suggestion
```

  - Richard Neher
  - Trevor Bedford
authorLinks:
  - https://twitter.com/sidneymbell
  - https://bedford.io/team/nicola-mueller/
  - https://bedford.io/team/cassia-wagner/
  - https://neherlab.org/emma-hodcroft.html/
  - https://bedford.io/team/james-hadfield/
  - https://neherlab.org/richard-neher.html/
  - https://bedford.io/team/trevor-bedford/
affiliations: "Fred Hutch, Seattle, USA; Biozentrum, Basel, Switzerland; CZI, CA, USA"
translators:
translatorLinks:
license: "CC-BY"  
licenseLink: "https://creativecommons.org/licenses/by/4.0/"
dataset: "https://nextstrain.org/ncov/europe/2020-04-30?d=map&f_region=Europe&legend=closed"

> 

Reviewer 2: ```suggestion
date: 2020-May-01
```

<!-- This is left-side text -->
# [Table of Contents](https://nextstrain.org/ncov/europe/2020-04-30?d=map&f_region=Europe&legend=closed)

* [Background resources](https://nextstrain.org/narratives/ncov/sit-rep/2020-XX-XX?n=2).     
* [About this data](https://nextstrain.org/narratives/ncov/sit-rep/2020-XX-XX?n=3).
* [Updates for Russia](https://nextstrain.org/narratives/ncov/sit-rep/2020-XX-XX?n=X).
* [Updates for Turkey](https://nextstrain.org/narratives/ncov/sit-rep/2020-XX-XX?n=X).
* [Updates for the Czech Republic](https://nextstrain.org/narratives/ncov/sit-rep/2020-XX-XX?n=X).
* [Updates for Poland](https://nextstrain.org/narratives/ncov/sit-rep/2020-XX-XX?n=X).
* [Data from Hungary, Slovakia, Slovenia and Latvia](https://nextstrain.org/narratives/ncov/sit-rep/2020-XX-XX?n=X).
* [What you can do](https://nextstrain.org/narratives/ncov/sit-rep/2020-XX-XX?n=X).
* [FAQ & common misconceptions](https://nextstrain.org/narratives/ncov/sit-rep/2020-XX-XX?n=X).
* [Scientific credit](https://nextstrain.org/narratives/ncov/sit-rep/2020-XX-XX?n=X).

<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
# Executive summary

> We analyzed 4,442 publicly shared COVID-19 genomes. By comparing these viral genomes to each other, we can characterize how COVID-19 is moving around the world and spreading locally. In this week's updates, we report:

Reviewer 2: ```suggestion
We analyzed 4,442 publicly shared COVID-19 genomes. By comparing these viral genomes to each other, we can characterize how COVID-19 is moving around the world and spreading locally. In this week's updates, we focus on Central & Eastern Europe and report:
```

* Many introductions to Russia  
* Heterogeneous outbreak in Moscow  
* Majority of St. Petersburg outbreak tied to a single introduction, most likely from Western Europe  
* Extensive mixing of cases across Turkey following repeated travel-related introductions  
* At least 5 local transmission chains in the Czech Republic, mostly likely due to repeated introductions from Western Europe
* At least 11 separate introductions to Poland
* Early samples from Hungary, Latvia, Poland, Slovakia and Slovenia  
```

<!-- ############ SLIDE BREAK ############# -->

<!-- This is left-side text 2-->
# [COVID-19 Resources](https://nextstrain.org/ncov/europe/2020-04-30?d=tree&f_region=Europe&legend=closed)
We've prepared some resources that will make interpreting the data we present in this narrative easier.
#### Nextstrain Resources  
* [START HERE: How to read a phylogeny](https://nextstrain.org/narratives/trees-background/).  
* [Background on coronaviruses](https://nextstrain.org/help/coronavirus/human-CoV).
* [Common misconceptions](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-13?n=11).

#### External Resources  

> * [Intuition: how SARS-CoV-2 mutates and spreads](https://www.nytimes.com/interactive/2020/04/30/science/coronavirus-mutations.html).

Reviewer 2: ```suggestion
* [How SARS-CoV-2 mutates and spreads](https://www.nytimes.com/interactive/2020/04/30/science/coronavirus-mutations.html).
```
Where did the Intuition come frome? It's not in the current title.

> * [Ask a Scientist & FAQs](https://covid19.fas.org/l/en).

Reviewer 2: ```suggestion
* [Ask a Scientist & FAQs](https://covid19.fas.org/).
```
Broken link

* [WHO Situation Reports](https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports).
* [CDC Resources](https://www.cdc.gov/coronavirus/2019-ncov/index.html).

> * [Making sense of uncertainty](https://www.theatlantic.com/health/archive/2020/04/pandemic-confusing-uncertainty/610819/).

Reviewer 2: This title is also different. I think we should use the titles of articles if we link to them.

> * [Making sense of uncertainty](https://www.theatlantic.com/health/archive/2020/04/pandemic-confusing-uncertainty/610819/).

Author: ```suggestion
* [Why the coronavirus is so confusing](https://www.theatlantic.com/health/archive/2020/04/pandemic-confusing-uncertainty/610819/).
```

Highlighting samples from Russia, we see that they are scattered both geographically (across Russia) and genomically (across the tree).
<br><br>
This indicates that the epidemic in Russia was sparked by many separate introductions, rather than a single introduction followed by within-country spread.
<br><br>
While there are a few samples towards the bottom of the tree that are most closely related to Asian cases, the majority of Russian samples descend from European lineages.

<!-- There is no right side text -->

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text -->
# [Heterogeneous outbreak in Moscow](https://nextstrain.org/ncov/europe/2020-04-29?c=division&d=tree&f_country=Russia&f_division=Moscow&label=clade:A2&p=full&r=division&m=div)

An example of frequent importation can be seen by zooming in on samples from Moscow.
<br><br>
There are a few clusters of cases which are most closely related to other samples from Moscow, indicating likely local transmission.
However, most samples from Moscow are most closely related to samples from elsewhere, consistent with importation.
<br><br>

> Given the number of confirmed cases in Moscow, it's likely that at least some of these introductions led to further local spread, but we don't yet have enough data to see this.

Reviewer 2: How many cases are there in Moscow currently? Would be good to add in this in, like we did last week.

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text -->

> # [Large outbreak cluster in St. Petersburg, sparked in early March](https://nextstrain.org/ncov/europe/2020-04-29?c=division&d=tree&f_country=Russia&f_division=Saint-Petersburg&f_region=Europe&label=clade:A2&p=full&r=division)

Reviewer 3: Would say that there are many introductions causing the outbreak, some sampling bias could also explain St. Petersburg only cluster

Author: Noted in the text :)

We can look at St. Petersburg as a counterexample.
<br><br>
As with Moscow, there are samples from St. Petersburg across the tree, indicating that there have been multiple introductions.

> Contrastingly, if you 'Explore the Data Yourself' (top right) and zoom into the main cluster in the center of this view, we can see a large clade that is made up entirely of samples from St. Petersburg.

Reviewer 2: ```suggestion
Contrastingly, if you click 'Explore the Data Yourself' and zoom into the main cluster in the center of this view, we can see a large clade that is made up entirely of samples from St. Petersburg.
```
To make consistent for mobile view.

<br><br>
This pattern is consistent with one of these introductions to St. Petersburg taking off to spark widespread transmission across the city, starting around the first week of March. This large outbreak cluster is most closely related to samples from Western Europe, although we cannot rule out the possibility of an unsampled, intermediate transmission from elsewhere.
<!-- There is no right side text -->

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text -->
# [Five local transmission chains in the Czech Republic](https://nextstrain.org/ncov/europe/2020-04-29?d=tree,map&f_country=Czech%20Republic&m=div&p=full)

The data from the Czech Republic show a mix of these two patterns.
<br><br>
There are at least 7 instances where a sample isolated in the Czech Republic descends from a virus that was mostly likely circulating elsewhere in Europe, indicating new introductions between March 1 and April 2.
<br><br>
For 5 of these introductions, we also see closely related local cases, sampled over time. This is consistent with local transmission.
<br><br>

> From this, we can say that there were at least 7 introductions to the Czech Republic; at least 5 of these likely led to further local transmission.

Reviewer 2: ```suggestion
From this, we can say that there were at least 7 introductions to the Czech Republic, at least 5 of these likely led to further local transmission.
```

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text -->
# [Extensive mixing of cases across Turkey](https://nextstrain.org/ncov/europe/2020-04-29?c=country_exposure&d=tree,map&f_country=Turkey&label=clade:A2&p=full&r=division&legend=open)

In the middle of the tree, we see a string of cases sampled from many different provinces around Turkey.
Some of these cases have reported travel history to Saudi Arabia, with subsequent closely related cases in Turkey.
<br><br>
Normally, we might assume that returning travelers had passed the virus to household members or neighbors back home.
Interestingly, though, the travelers' cases were often sampled from different provinces than the most closely related cases.
<br><br>

> This indicates that there has been extensive mixing of viral lineages, reaching many provinces of Turkey within one week.

Reviewer 2: Or multiple travelers from Saudi.

We may also speculate that a large gathering could explain these observations, although we would need additional epidemiological data to further investigate.
<!-- There is no right-side text -->

<!-- There is no right-side text -->
<!-- ############ SLIDE BREAK ############# -->

<!-- This is left-side text -->
# [At least 11 introductions to Poland](https://nextstrain.org/ncov/europe/2020-04-29?d=tree,map&f_country=Poland&p=full)

We have 15 sequences from Poland on Nextstrain. Of these 15, at least 11 of them represent separate introductions from elsewhere in the world.
As in Moscow, given the number of reported cases, we might expect that several of these introductions have seeded local transmission chains in Poland, but we can't yet see this in the genomic data.
<!-- There is no right-side text -->

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text -->

> # [Early samples from Hungary, Latvia, Slovakia and Slovenia](https://nextstrain.org/ncov/europe/2020-04-29?d=tree,map&f_country=Hungary,Latvia,Slovakia,Slovenia&f_region=Europe&p=full)

Reviewer 3: Would remove section, since we don't really say anything about transmission

Author: @nicfel @cassiawag -- I'm a little torn here. I threw this in just to acknowledge that we have data from these areas and think it's important, even though we can't yet say much about it. Votes on whether to keep it in or remove it?

Reviewer: I'm ok with it myself, I think it's fair to recognise the effort. But no strong feelings.
